ts/Series/Funnel3D/Funnel3DSeries.ts

```diff
@@ -257,16 +256,16 @@ class Funnel3DSeries extends ColumnSeries {
         extend(this.xAxis.options, {
             gridLineWidth: 0,
             lineWidth: 0,
-            title: null,
+            title: void 0,
             tickPositions: []
-        });
-        extend(this.yAxis.options, {
+        } as DeepPartial<Highcharts.AxisOptions>);
+        merge(true, this.yAxis.options, {
             gridLineWidth: 0,
-            title: null,
+            title: void 0,
             labels: {
                 enabled: false
             }
-        });
+        } as DeepPartial<Highcharts.AxisOptions>);
```

Comment:
The `something as somethingElse` is forced typing and reduces the type check to the primitive scope, in the sense that return type and forced type share the same primitive type (object).

Reply:
Yes, we should have the merge return the same type as its first argument, (or second argument if the first is bool).

Reply:
Alright, this _does_ work. The `as` was probably just needed for an intermediate step.